SecurityPkg/SecurityPkg.dec

```diff
@@ -520,6 +520,18 @@
   # @Prompt TPM device address.
   gEfiSecurityPkgTokenSpaceGuid.PcdTpmBaseAddress|0xFED40000|UINT64|0x00010012
 
+    ## This PCD indicates internal TPM base address.<BR><BR>
+  # @Prompt TPM device base address.
+  gEfiSecurityPkgTokenSpaceGuid.PcdTpmInternalBaseAddress|0x10000200000|UINT64|0x0001002A

```

Comment:
small nit: I understand that the other PCDs are out of identifier order, but can we add new ones in order? Makes it easier to add new PCDs in the future and not have to figure out what the next identifier is.